helpers/discovery/getDiscovery.ts

```diff
@@ -11,7 +21,7 @@ export async function getDiscovery(bot: Bot, guildId: bigint) {
 
   return {
     guildId,
-    primaryCategoryId: result.primary_category_id,
+    primaryCategoryId: BigInt(result.primary_category_id),
```

Comment:
Confirm this is indeed a snowflake and if it is use the transformers.snowflake

Reply:
On it.

Reply:
After some digging: It is not a snowflake. Just a number. Fixed in latest commit.